rllib/policy/eager_tf_policy.py

```diff
@@ -175,22 +175,39 @@ def compute_actions_from_input_dict(
         ) -> Tuple[TensorType, List[TensorType], Dict[str, TensorType]]:
             """Traced version of Policy.compute_actions_from_input_dict."""
 
-            # NOTE: In the new RLModule stack the sampling side is not traced with this
-            # justification that in order to speed up sampling we need to use more
-            # actors.
             # Create a traced version of `self._compute_actions_helper`.
-            if (
-                not self.config.get("_enable_rl_module_api", False)
-                and self._traced_compute_actions_helper is False
-                and not self._no_tracing
-            ):
-                self._compute_actions_helper = _convert_eager_inputs(
-                    tf.function(
-                        super(TracedEagerPolicy, self)._compute_actions_helper,
-                        autograph=False,
-                        reduce_retracing=True,
+            if self._traced_compute_actions_helper is False and not self._no_tracing:
```

Comment:
Had to separate these out as forward_inference and forward_explore return different structures, which breaks tf2-tracing.